ai-ml/emr-spark-rapids/addons.tf

```diff
@@ -46,16 +28,18 @@ module "eks_blueprints_kubernetes_addons" {
   # Amazon EKS Managed Add-ons
   #---------------------------------------
   eks_addons = {
-    # NOTE: If you are using a custom VPC CNI, you must disable the VPC CNI add-on before you can delete the custom VPC CNI.
-    # VPC CNI should be installed before the worker nodes are created for custom VPC CNI to work properly.
-    vpc-cni = {
-      service_account_role_arn = module.vpc_cni_irsa.iam_role_arn
-    }
     aws-ebs-csi-driver = {
       service_account_role_arn = module.ebs_csi_driver_irsa.iam_role_arn
     }
-    coredns    = {}
-    kube-proxy = {}
+    coredns = {
```

Comment:
standardized on this configuration across the board. Let me know if we want to tweak this and I can update. I think at minimum, keeping the preserve on the CNI is helpful for the examples